### ISSUE TITLE
Error apt-upgrading 6.13.0-1 to 6.13.0-2

### ISSUE THREAD

**Issue by peci1** [gazebosim/gz-math#514]
## Environment
* OS Version: Ubuntu 18.04
* Source or binary build? Binary, upgrade from 6.13.0-1 to 6.13.0-2

During apt upgrade, I got this error:

```
Unpacking libignition-math6-eigen3-dev:amd64 (6.13.0-2~bionic) over (6.13.0-1~bionic) …
dpkg: error processing archive /var/cache/apt/archives/libignition-math6-eigen3-dev_6.13.0-2~bionic_amd64.deb (--unpack):
 trying to overwrite file „/usr/include/ignition/math6/gz/math/eigen3/Conversions.hh“, which is also in package libignition-math6-dev:amd64 6.13.0-1~bionic

Unpacking libignition-math6:amd64 (6.13.0-2~bionic) over (6.13.0-1~bionic) …
There were errors while processing the changes:
 /var/cache/apt/archives/libignition-math6-eigen3-dev_6.13.0-2~bionic_amd64.deb

```

**Comment by scpeters** [comment]
I rebuilt the debs after https://github.com/gazebo-release/gz-math6-release/pull/19 because some eigen headers were not in the right package, but now I saw that same error when upgrading. I think an `apt install -f` fixed it on my machine, but I'm not sure. I opened https://github.com/gazebo-release/gz-math6-release/pull/20 to see if it would fix the upgrade issue, but it's the weekend and I haven't had a change to get any feedback on whether it would work. I can make a test build from that branch if someone will test it

**Comment by peci1** [comment]
Thanks for confirming. Apt install -f also fixed it for me.

**Comment by scpeters** [comment]
I just started a test build for bionic that will upload to the prerelease repo: https://github.com/gazebo-release/gz-math6-release/pull/20#issuecomment-1279849163

I'm not sure if you can reproduce it anymore after the `apt install -f`

**Comment by peci1** [comment]
Yes, I'm also afraid this won't be easily reproducible at my side. Unfortunately, I only had one machine with 6.13.0-1 and that's already upgraded. Clean install works.

**Comment by scpeters** [comment]
> I just started a test build for bionic that will upload to the prerelease repo: [gazebo-release/gz-math6-release#20 (comment)](https://github.com/gazebo-release/gz-math6-release/pull/20#issuecomment-1279849163)
> 
> I'm not sure if you can reproduce it anymore after the `apt install -f`

this PR seems to help, so I've merged it and am rebuilding 6.13.0-3 debs

**Comment by scpeters** [comment]
I've rebuilt the debs, so I think this is fixed. Please comment or reopen if the issue persists
